### DISCUSSION TITLE
Windows installation warning

### DISCUSSION THREAD

**Discussion by rutschi100** [massCodeIO/massCode#188] [Ideas]
During the installation, Windows reports with a warning. Would it be possible to register the application so that Windows does not consider it a potential danger?

I don't know that much about it. Would it be a very big effort? I think that without this registration in many companies, the application may not be installed.

![grafik](https://user-images.githubusercontent.com/50636192/182832773-b16f6f76-3720-439e-9916-020941a69520.png)
![grafik](https://user-images.githubusercontent.com/50636192/182832783-a8300670-904c-48e9-ab3b-acf88d6576f2.png)

**Comment by antonreshetov** [comment]
This is the usual behavior of the Windows Defender because the application does not have a signature. Unfortunately I am not able to pay for Windows Authenticode or the Apple Developer Program at the moment.

**Reply by rutschi100** [reply]
Oh, okay. I googled it right away. For open-source projects, it costs 60+ /year.

I hope the donations will allow it at some point. As soon as I have some income again, I will participate.
